source/base/Profiler.ooc

```diff
@@ -65,6 +65,7 @@ Profiler: class {
 	}
 	resetAll: static func { This _profilers apply(func (profiler: This) { profiler reset() }) }
 	dispose: static func {
+		This printResults()
```

Comment:
Since that's the only change in this PR we should close this then. Any objections @ziquanyu ?